src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue

```diff
@@ -2,25 +2,25 @@
   <UiListItem
     :id="id"
     :tag="component"
+    :value="value"
+    :model-value="modelValue"
+    :text-label-attrs="defaultProps.textLabelAttrs"
+    :suffix-attrs="defaultProps.suffixAttrs"
     :class="[
       'ui-multiple-answer-item', errorClass
     ]"
-    :value="value"
-    :model-value="modelValue"
-    v-bind="defaultProps"
```

Comment:
IMO 
```
:text-label-attrs="defaultProps.textLabelAttrs"
 :suffix-attrs="defaultProps.suffixAttrs"
```
is more redable than `defaultProps`. You know what you bind. I can find value for use `v-bind="defaultProps"`